sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -860,6 +864,24 @@ abstract class CatalogTestUtils {
       bucketSpec = Some(BucketSpec(4, Seq("col1"), Nil)))
   }
 
+  def newView(
+      name: String,
+      database: Option[String] = None): CatalogTable = {
+    val viewDefaultDatabase = database.getOrElse("default")
+    CatalogTable(
+      identifier = TableIdentifier(name, database),
+      tableType = CatalogTableType.VIEW,
+      storage = CatalogStorageFormat.empty,
+      schema = new StructType()
+        .add("col1", "int")
+        .add("col2", "string")
+        .add("a", "int")
+        .add("b", "string"),
+      viewOriginalText = Some("SELECT * FROM tbl1"),
+      viewText = Some("SELECT * FROM tbl1"),
+      properties = Map[String, String] {CatalogTable.VIEW_DEFAULT_DATABASE -> viewDefaultDatabase})
```

Comment:
nit: `Map(CatalogTable.VIEW_DEFAULT_DATABASE -> viewDefaultDatabase)`, scala comiler will infer the type for us